receiver/awscontainerinsightreceiver/cadvisor/cadvisor_linux.go

```diff
@@ -0,0 +1,35 @@
+// Copyright  OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// +build linux
+
+package cadvisor
```

Comment:
Can we expose less public API by including in awsinsightreceiver package?

Reply:
I am not sure I understand what you mean. Are you saying that we should put the relevant file like `cadvisor_linux.go` inside `awscontainerinsightreceiver` package so that we don't need to expose any public API here?

Reply:
Yeah. Or alternatively into the internal package, but as it is right now, these seem like public APIs usable outside this component.

Reply:
That makes sense. I will put the code inside an internal folder